adafruit_lsm303_accel.py

```diff
@@ -394,3 +410,42 @@ def _read_bytes(device, address, count, buf):
         with device as i2c:
             buf[0] = address & 0xFF
             i2c.write_then_readinto(buf, buf, out_end=1, in_end=count)
+
+    @property
+    def fifo_mode(self):
+        """Sets the fifo mode of the sensor. The mode must be a `FifoMode`."""
+        return self._cached_fifo_mode
+
+    @fifo_mode.setter
+    def fifo_mode(self, value):
+        if value < 0 or value > 2:
+            raise AttributeError("fifo mode must be a 'FifoMode'")
+        if value == 4:
+            self._fifo_enable = 0
+        else:
+            self._fifo_enable = 1
+            self._fifo_mode = value
+            self._cached_fifo_mode = value
+
+    def _raw_fifo(self):
+        self._read_bytes(
+            self._accel_device, _REG_ACCEL_OUT_X_L_A | 0x80, 192, self._FIFO_BUFFER
+        )
+        raw_data = []
+        for i in range(0, 192, 6):
+            raw_data.append(struct.unpack_from("<hhh", self._FIFO_BUFFER[i : i + 6]))
+        return raw_data
+
+    def fifo_acceleration(self):
```

Comment:
This could be a property. I think that would help keep it consistent with the existing `acceleration` property.